extras.php

```diff
@@ -1,12 +1,12 @@
 <?php
-
 /**
- * Extra File where a lot of the extra functions from plugin.php go.
+ * Extra File
+ *
+ * Contains extra functions from plugin.php go.
  *
+ * @since 4.4.0
```

Comment:
I added this, since the other function in this file had the same @since tag. Let me remove that.